rimage/manifest.c

```diff
@@ -124,9 +124,16 @@ static uint32_t elf_to_file_offset(struct image *image,
 				   struct sof_man_module *man_module,
 				   Elf32_Shdr *section)
 {
-	uint32_t elf_addr = section->vaddr, file_offset = 0;
+	uint32_t elf_addr = section->vaddr, file_offset = 0, i;
 
 	if (section->type == SHT_PROGBITS) {
+		/* check programs for lma/vma change */
+		for (i = 0; i < module->hdr.phnum; i++) {
+			if (section->vaddr == module->prg[i].vaddr) {
+				elf_addr = module->prg[i].paddr;
+				break;
```

Comment:
Can we have a flag for each section here where we make this LMA/VMA change. This flag would be used to diplay either a "L" or "V" next to section addresses in the ELF section dump (as part of the build output).

Reply:
Are we getting this flag in the output ? It would be confusing otherwise to see some addresses being different than expected.

Reply:
@lgirdwood yes we can, Im thinking how to do it best way.
In general, this is not section attribute in elf. Its available in programs and we already can show them (they are verbose print currently).

Im not sure if "L" or "V" will be okay because most of the sections have equal LMA and VMA. So maybe display "LV" for most sections when they are equal and "L" or "V" only when we are printing only one of them?

If yes, we should print LMA where they actually are or VMA like now?

Reply:
> Im not sure if "L" or "V" will be okay because most of the sections have equal LMA and VMA. So maybe display "LV" for most sections when they are equal and "L" or "V" only when we are printing only one of them?

I think we only care if we have the lma/vma change, i.e. if there is no change then we dont need to flag the output, if there is change we could label V or L ?

> 
> If yes, we should print LMA where they actually are or VMA like now?

We could print both, you could change one of the column headers and add a new column. The main this is that it's obvious which one is being used.

The main thing is to help the user understand the layout when debugging. Adding this information will really help.

Reply:
> We could print both, you could change one of the column headers and add a new column. The main this is that it's obvious which one is being used.

Ok, I have changed section view to show both of them, in this case we dont need flags, right?
Please check attached sample file.
![alt](https://user-images.githubusercontent.com/45165721/77301921-c3bb3c00-6cf0-11ea-8318-b76fda6dce4b.png)